components/rowListContent.brs

```diff
@@ -2,10 +2,21 @@ sub init()
     m.global.http.request = { payload: {
             userId: 1,
             title: "WOW, IT WORKED ^_^",
-    }, url: "https://jsonplaceholder.typicode.com/albums", requestType: "GET"}
-    m.global.http.observeFieldScoped("response", "useResponse")
+    }, url: "https://alghool.net/RokuDev/data.json", requestType: "GET"}
+    m.global.http.observeFieldScoped("response", "populateRowList")
 end sub
 
-sub useResponse()
-    ?m.global.http.response
+sub populateRowList()
+    items = m.global.http.response.body
+    for each item in items
+        itemNode = m.top.createChild("ContentNode")
+        itemNode.title = item.title
+        for each data in item.data
+            itemData = itemNode.createChild("ContentNode")
+            itemData.title = data.title
+            itemData.HDPosterUrl = data.thumbnail
+            itemData.description = data.longDescription
```

Comment:
```suggestion
            itemData. setFields({
                title: data.title
                HDPosterUrl: data.thumbnail
                description: data.longDescription
            })
```

Reply:
setFields passing in an object where keys are the field ids, noted.